ledger-core/virtual/execute/execute.go

```diff
@@ -235,13 +235,14 @@ func (s *SMExecute) stepWaitObjectReady(ctx smachine.ExecutionContext) smachine.
 			panic(throw.NotImplemented())
 		}
 	} else if objectState != object.HasState {
-		panic(throw.E("no state on object after readyToWork", struct {
+		s.prepareExecutionError(throw.E("object is not ready", struct {
```

Comment:
I don't like the error message. It doesn't make a lot of sense to the user. Maybe something like "object not found" or "object does not exist" is better?

Reply:
@sexyhamster what do you think about this error message?

Reply:
"object does not exist" sounds OK, maybe later on there will be another case for deactivated object

Reply:
I would disappoint you and say that this is not correct place to check for anything but Missing. For example if in this point of code the state is Inactive then it doesn't mean that request should fail. The request can be a duplicate and should be finished. Previous panic is not correct and should have been panic(NotImplemented). I insist on returning this new particular error only when `objectState == object.Missing`

Reply:
@ruz  is this correct ? https://github.com/insolar/assured-ledger/pull/400/commits/07eb4c04a21d51116f982474e30b2524db5b5053

Reply:
yep, way better. `make generate`

just in case people wonder

- Empty - may happen on recursive call while constructor is running
- Inactive - may happen in this place for two reasons: call on inactive object and duplicate of old request